dot/parachain/provisioner/provisioner.go

```diff
@@ -0,0 +1,72 @@
+package provisioner
+
+import (
+	"context"
+	"time"
+
+	provisionermessages "github.com/ChainSafe/gossamer/dot/parachain/provisioner/messages"
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	"github.com/ChainSafe/gossamer/internal/log"
+	"github.com/ChainSafe/gossamer/lib/common"
+)
+
+var logger = log.NewFromGlobal(log.AddContext("pkg", "provisioner"))
+
+// we expect an inherent to be ready after this time.
+const INHERENT_TIMEOUT = time.Millisecond * 2000
```

Comment:
Can we define this as seconds?
```suggestion
const InherentTimeout = 2 * time.Second
```